### PR TITLE
Feature/minor additions

### PR DESCRIPTION
**Pull request by robmoss**
Add a few tips/info blocks:

- Advocate for acknowledging peer code reviewers in papers;
- Link to tools that improve working with Git, particularly for Windows users; and
- Advising contributors to write one sentence per line.

### CODE REVIEW

**Comment by robmoss** [comment]
Also added a link to a PowerShell tutorial, in case there are Windows users who prefer to avoid Bash.

File: src/how-to-contribute.md [hunk 1]
```diff
@@ -31,6 +31,12 @@ You can also edit any page by clicking the "Suggest an edit" button (<i class="f
 This will start the process described above by forking the book repository.
 ```
 
+```admonish tip
+When editing Markdown content, please **write each sentence on a separate line**.
```
Comment (EamonConway) [inline src/how-to-contribute.md]: Should we consider rewording this to 
When editing Markdown content, please start each sentence on a new line?
Feel like this is less ambiguous

**Review by EamonConway — CHANGES_REQUESTED** [review]
I'm more than happy for this to be merged into master. There is just one suggestion that I have made to make a sentence structure more clear in markdown. I felt the current phrasing was confusing.

File: src/how-to-contribute.md (hunk 1 above)
Comment (robmoss) [reply src/how-to-contribute.md]: Yep, that makes it clear. Good point!

Comment (robmoss) [reply src/how-to-contribute.md]: I'm trying to figure out if the UI can create the commit for us, or if I need to do it myself ...

**Pull request merged**